multitier/finders.py

```diff
@@ -45,12 +53,45 @@ def __init__(self, app_names=None, *args, **kwargs):
         if site is not None:
             # ``site`` could be ``None`` when this code is used through
             # a manage.py command (ex: collectstatic).
-            roots += [os.path.join(django_settings.STATIC_ROOT,
-                theme, postfix) for theme in get_current_site().get_templates()]
-        roots += [os.path.join(django_settings.STATIC_ROOT, postfix)]
+            roots += [os.path.join(static_dir,
+                theme, postfix) for static_dir in STATICS_DIRS
+                    for theme in get_current_site().get_templates()]
+        roots += [os.path.join(static_dir, postfix)
```

Comment:
Should we really add those here? Couldn't rely on a StaticFinder later down the chain of ``STATICFILES_FINDERS``?